injector/dns_disruption.go

```diff
@@ -134,40 +134,27 @@ func (i *DNSDisruptionInjector) Inject() error {
 	}
 
 	// Set up iptables rules
-	if err := i.config.Iptables.CreateChain(chaostypes.InjectorIptablesChaosDNSChainName); err != nil {
-		return fmt.Errorf("unable to create new iptables chain: %w", err)
-	}
-
-	if err := i.config.Iptables.AddRuleWithIP(chaostypes.InjectorIptablesChaosDNSChainName, "udp", "53", "DNAT", podIP); err != nil {
+	if err := i.config.Iptables.RedirectTo("udp", "53", podIP); err != nil {
```

Comment:
Would it be possible to add a little more context to this comment? Not sure to understand what kind of rules are set up here

Reply:
Sure, will do that